examples/flex/main.go

```diff
@@ -36,10 +36,8 @@ func NewModel() Model {
 				columnKeyDescription: "Super zappy mouse, handle with care",
 			}),
 			table.NewRow(table.RowData{
-				columnKeyName:    "Charmander",
-				columnKeyElement: "Fire",
-				// TODO: Fix double width string length limiting!
-				//columnKeyDescription: "直立した恐竜のような身体と、尻尾の先端に常に燃えている炎が特徴。",
+				columnKeyName:        "Charmander",
+				columnKeyElement:     "Fire",
 				columnKeyDescription: "Lots of fire",
```

Comment:
```suggestion
				columnKeyDescription: "直立した恐竜のような身体と、尻尾の先端に常に燃えている炎が特徴。",
```

I'd like to keep this example in place to show how it truncates wide characters.